lockbox-iosTests/RouteActionSpec.swift

```diff
@@ -130,19 +124,33 @@ class RouteActionSpec: QuickSpec {
                 it("event object is equal to the login view shown") {
                     expect(LoginRouteAction.welcome.eventObject).to(equal(TelemetryEventObject.loginWelcome))
                     expect(LoginRouteAction.fxa.eventObject).to(equal(TelemetryEventObject.loginFxa))
-                    expect(LoginRouteAction.learnMore.eventObject).to(equal(TelemetryEventObject.loginLearnMore))
                 }
 
                 it("returns nil as the value") {
                     expect(LoginRouteAction.welcome.value).to(beNil())
                     expect(LoginRouteAction.fxa.value).to(beNil())
-                    expect(LoginRouteAction.learnMore.value).to(beNil())
                 }
 
                 it("returns nil as for the extras") {
                   expect(LoginRouteAction.welcome.extras).to(beNil())
                   expect(LoginRouteAction.fxa.extras).to(beNil())
-                  expect(LoginRouteAction.learnMore.extras).to(beNil())
+                }
+            }
+        }
+
+        describe("ExternalWebsiteRouteAction") {
+            describe("equality") {
+                it("actions are equal when the title and URL string are equal") {
+                    expect(ExternalWebsiteRouteAction(urlString: "www.butts.com", title: "blah", returnRoute: MainRouteAction.list))
```

Comment:
I think a less ... controversial ... URL ought to be used herein.

Maybe "cats.example" or "cats.com"?